mavros_msgs/msg/Debug.msg

```diff
@@ -0,0 +1,26 @@
+# Msg for Debug MAVLink API
+#
+# Supported types:
+# DEBUG			http://mavlink.org/messages/common#DEBUG
+# DEBUG_VECTOR			http://mavlink.org/messages/common#DEBUG_VECT
+# NAMED_VALUE_FLOAT		http://mavlink.org/messages/common#NAMED_VALUE_FLOAT
+# NAMED_VALUE_INT		http://mavlink.org/messages/common#NAMED_VALUE_INT
+# @TODO: add support for DEBUG_ARRAY (https://github.com/mavlink/mavlink/pull/734)
+
+# debug types for msg
+uint8 DEBUG_TYPE		= 0
+uint8 DEBUG_VECT_TYPE		= 1
+uint8 NAMED_VALUE_FLOAT_TYPE	= 2
+uint8 NAMED_VALUE_INT_TYPE	= 3
```

Comment:
Debug and Value-float is almost the same. Maybe we may reduce types? But not a problem.

Reply:
Also note: im prefer define enum values after field.

Reply:
Should be prefixed, `TYPE_DEBUG`, `TYPE_NAMED_VALUE_FLOAT`...

Reply:
sure